org.jacoco.core/src/org/jacoco/core/internal/Java9Support.java

```diff
@@ -41,9 +41,6 @@ private Java9Support() {
 	 */
 	public static byte[] readFully(final InputStream is)
 			throws IOException {
-		if (is == null) {
```

Comment:
@Godin Why do we need this check specifically here? I think all other APIs like `analyzeAll()` will result in a NPE for a `null` `InputStream`.  Which looks ok for me.